src/store/initialState.js

```diff
@@ -24,8 +24,8 @@ const initialState = {
   images: {},
 
   leaderboard: {
-    loading: false,
```

Comment:
I do not see why `loading` is no longer required in the initial state?

Reply:
Because we track loading locally within the table